crates/eips/src/eip7685.rs

```diff
@@ -77,6 +77,30 @@ impl Requests {
     }
 }
 
+/// A list of requests or a precomputed requests hash.
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub enum PragueRequests {
```

Comment:
please find a better name like `RequestsOrHash`, this is very vague

Reply:
additionally, we should note that the hash is for testing purposes only, so please add more doc comments on why this enum exists